src/main/resources/grammar.hgr

```diff
@@ -329,7 +329,7 @@ grammar {
     $call_input = :input :colon list($mapping, :comma) -> Inputs(map=$2)
     $mapping = :identifier :equal $e -> IOMapping(key=$0, value=$2)
     $alias = :as :identifier -> $1
-    $wf_outputs = :output :lbrace list($wf_output, :comma) :rbrace -> WorkflowOutputs(outputs=$2)
+    $wf_outputs = :output :lbrace list($wf_output) :rbrace -> WorkflowOutputs(outputs=$2)
```

Comment:
Is Green aware of this change?  I know at least one person there was writing output sections even though Cromwell wasn't honoring them.